Task_4/scripts/app.js

```diff
@@ -0,0 +1,32 @@
+const moduleTasks = (function(){
+    const user = 'Julia Grib';
+
+    function isValidateId(id){
+        return typeof id === 'string' ? true : false;
```

Comment:
typeof id === 'string'  return boolean    . In this case ternar operator don't need

Reply:
isValidateId rename to isValidTypeId